modules/restful_example/plugins/restful/db_query/watchdog/1.0/watchdog.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$plugin = array(
+  'label' => t('Watchdog entries'),
+  'resource' => 'watchdog',
+  'name' => 'watchdog',
+  'data_provider_options' => array(
+    'table_name' => 'watchdog',
+    'id_column' => 'wid',
+  ),
+  'description' => t('Expose watchdog entries to the REST API.'),
+  'class' => 'RestfulWatchdogResource',
+  'authentication_types' => TRUE,
+  'authentication_optional' => TRUE,
```

Comment:
Actually we should add an access check to this endpoint, since it can contain sensitive information. We wouldn't want a security issue because someone left `restful_example` enabled.